lib/database/game.php

```diff
@@ -420,10 +420,10 @@ function getGamesListByDev($dev, $consoleID, &$dataOut, $sortBy, $ticketsFlag =
             break;
 
         case 6:
-            $query .= "ORDER BY gd.ConsoleID, ach.DateModified DESC, Title ";
+            $query .= "ORDER BY gd.ConsoleID, MAX(ach.DateModified) DESC, Title ";
```

Comment:
Good find 👍 

As @Jamiras pointed out this could be done by reusing the `DateModified` column alias, too, to keep it simpler.

See https://github.com/RetroAchievements/RAWeb/blob/master/lib/database/game.php#L357

Maybe a better name for that alias would be fitting - like `SetDateModified` as it's about the last change in a game's achievement set rather than a game's own metadata. Not that important, though.